pkg/integrations/manager.go

```diff
@@ -171,17 +174,17 @@ func newManager(c ManagerConfig, logger log.Logger, im instance.Manager, integra
 		}
 	}
 
-	go m.run(ctx)
+	go m.run(ctx, config)
```

Comment:
Can we store this in the integrations manager rather than passing it around through all the functions?